components/script/dom/webglrenderingcontext.rs

```diff
@@ -83,6 +83,8 @@ pub struct WebGLRenderingContext {
     bound_buffer_array: MutNullableHeap<JS<WebGLBuffer>>,
     bound_buffer_element_array: MutNullableHeap<JS<WebGLBuffer>>,
     current_program: MutNullableHeap<JS<WebGLProgram>>,
+    #[ignore_heap_size_of = "Because it's small"]
+    current_vertex_attrib_0: Cell<(f32, f32, f32, f32)>,
```

Comment:
@dzbarsky Do you remember why it's necessary to keep the value of vertex attrib 0 on the DOM side?

Reply:
When I first implemented this in https://github.com/servo/servo/commit/2bf016ff149da46f8957b959a61d5c5a6afb682a `GetVertexAttrib` had a path to serve 0/CURRENT_VERTEX_ATTRIB from DOM side - I don't recall why I did this though...perhaps it was a fast path or something?

Reply:
Oh I see now that this is the original place we added it :) Most likely I was cargo-culting the pattern of retaining state like `current_program`, though this one was likely not necessary. Oops!

Reply:
Seems like calling `glGetVertexAttrib*` with index 0 causes issues on some drivers.

Reply:
https://www.khronos.org/webgl/wiki/WebGL_and_OpenGL_Differences#Vertex_Attribute_0